src/Talis/Persona/Client/Users.php

```diff
@@ -125,73 +144,61 @@ public function updateUser($guid, $profile, $token)
 
         try
         {
-            $user = $this->personaPatchUser($url, $profile, $token);
-            return $user;
+            return $this->performRequest(
+                $url,
+                array(
+                    'method' => 'PUT',
+                    'body' => json_encode($profile),
+                    'bearerToken' => $token,
+                )
+            );
         } catch(\Exception $e)
         {
-            throw new \Exception('User not updated');
+            $profileString = implode(',', $profile);
+            $this->getLogger()->error("Error updating user | guid: $guid | profile: $profileString | " . $e->getMessage());
+            throw new \Exception('Error updating user: ' . $e->getMessage());
         }
     }
 
-    /* Protected functions */
-
-    /**
-     * Get a persona user
-     * @param string $url
-     * @param string $token
-     * @param int    $cacheTTL time to live in seconds for cached responses
-     * @access protected
-     * @return mixed
-     * @throws \Exception
-     */
-    protected function personaGetUser($url, $token, $cacheTTL = 300)
-    {
-        return $this->performRequest(
-            $url,
-            array(
-                'bearerToken' => $token,
-                'cacheTTL' => $cacheTTL,
-            )
-        );
-    }
-
     /**
-     * Create a new user in Persona
-     * @param string $url
-     * @param array $query
+     * @param string $guid
+     * @param string $gupid
      * @param string $token
+     * @return array|null
+     * @throws \InvalidArgumentException
      * @throws \Exception
-     * @return array
+     * @access public
      */
-    protected function personaPostUser($url, $query, $token)
+    public function addGupidToUser($guid, $gupid, $token)
     {
-        return $this->performRequest(
-            $url,
-            array(
-                'method' => 'POST',
-                'body' => json_encode($query),
-                'bearerToken' => $token,
-            )
-        );
-    }
+        if(!is_string($guid) || trim($guid) === '')
+        {
```

Comment:
braces shouldn't be dropped for `if` statements, only functions and classes (PSR standards)